### DISCUSSION TITLE
Will Github add more achivements?

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#21206] [Other Feature Feedback, Questions, & Ideas]
I hope there will be a third release of achievements, they're so fun!

**Comment by radar-blip** [comment]
Of course.

**Comment by ilechuks73** [comment]
i hope they do.

**Reply by radar-blip** [reply]
They will, hopefully you will have to actually achieve something to get them.

**Comment by melvincwng** [comment]
@devkennyy , probably they would, hope to see some new exciting achievements in the future 👍🏻

**Reply by tvews8z** [reply]
> @devkennyy , probably they would, hope to see some new exciting achievements in the future 👍🏻

**Comment by ghost** [comment]
@devkennyy 
### I hope they don't add more, because it will rush a lot of people doing trivial things or doing nothing to get one.
### Proof of this is a large number of empty discussions to get an achievement
----
**But I do not deny that these achievements have pushed me forward,  to make a lot of contribution**

**Reply by devkennyy** [reply]
You're right but there will always be those kinds of people when there are achievements/badges. At this point most of the achievements aren't necessarily achievements but that's why I'm hoping they will add more. Ones that you can't get with the click of a button, ones that mean something.

**Comment by jfullstackdev** [comment]
as always, there are opposing views, but there is the ability to hide it at your Settings and more you can even set your profile to private

**Comment by aleksa-codes** [comment]
Mine won't show up even after changing the settings, I got one after a merge and after 2 weeks it is still not there. Any ideas, someone?

**Reply by jfullstackdev** [reply]
did you click `Update preferences` ?

**Reply by aleksa-codes** [reply]
Yes for both "Include private contributions on my profile" and "Show Achievements on my profile". Tried setting them off and on a couple of times, but still no luck. While both of those options are off I get a notification on my profile page to turn them on and display the new achievement, which I followed the first time.

**Reply by aleksa-codes** [reply]
Update: It was my bad, checked the requirements and the minimum is 2 merged PRs for the Pull Shark achievement.

**Comment by anhvin007** [comment]
yes

**Comment by SharfShiplo** [comment]
Yes they will, in this post they has mentioned about more achievements coming soon :  https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

**Comment by spenserblack** [comment]
I hope they add more "boolean achievements" instead of "numerical achievements."

Looking at video games, fun achievements are often "boolean," meaning you either did or didn't do it -- there isn't a "halfway done" point. For example, "beat the secret boss" is boolean, and "collect 1,000,000 gold" is numerical. Basically, if you have to "grind" for the achievement, it's not that fun IMO.

I'm not sure what that means for GitHub, but I think as an example "Contributed to a repository owned by GitHub", which is accessible and also not repetitive.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
